tests/flow/test_wcc.py

```
@@ -0,0 +1,387 @@
+                """CALL algo.WCC(null) YIELD node, invalidField""",  # non-existent yield field
+
+                """CALL algo.WCC({nodeLabels: ['Person'],
+                               relationshipTypes: ['KNOWS'],
+                               invalidParam: 'value'})""",           # unexpected extra parameters
+        ]
+
+        for q in invalid_queries:
+            try:
+                self.graph.query(q)
+                self.env.assertFalse(True)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)